scripts/provisioning/templates/s3.test.tmpl.1-node.sample

```diff
@@ -17,9 +17,5 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 endpoint_fqdn: "s3.seagate.com"
-cortx:
-  software:
-    openldap:
-      sgiam:
-        user: "sgiamadmin"
-        secret: "gAAAAABgWFPz2w_t6IBVictZmgTCgBXn5V-oy2JUVtvZkJvHM5KaaZKXRwtz4YKYieIy8mVKF8PUaZ6WYJEttcsi2KDpLOk9Vg=="
+openldapiamuser: "sgiamadmin"
```

Comment:
Why not put hierarchy :
cortx>software>openldap>sgiam>user
cortx>software>openldap>sgiam>password

Reply:
The only concern I have is how do we maintain the isolation?
Thats why I recommended a flat hirarchy @nitin-seagate and I think we should pursue the same if there is no strong reason for adding it to cortx.

Reply:
Lets keep the same hierarchy as I proposed, eventually even if we merge to ConfStore it would just mean over-ridding of values...